common/test/acceptance/pages/lms/discussion.py

```diff
@@ -132,6 +133,7 @@ def verify_mathjax_rendered(self):
 
     def is_response_visible(self, comment_id):
         """Returns true if the response is viewable onscreen"""
+        self.wait_for_ajax()
```

Comment:
Was causing failure in https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/discussion/test_discussion.py#L591

Since that test checks `is_comment_visible` before `is_comment_editable`, the `wait_for_ajax` protection was not in play.